app/src/main/java/com/hehc/roomsample/repository/StarshipRepositoryContract.kt

```diff
@@ -5,5 +5,6 @@ import com.hehc.roomsample.persistance.Starship
 interface StarshipRepositoryContract {
     suspend fun getAllStarships(): List<Starship>
     suspend fun saveStarship(ship: Starship)
+    suspend fun getAllFavorites(): List<Starship>
```

Comment:
Remember the Open-Closed principle. For larger projects you may want to create a V2 interface, and implement from there.